elytron-oidc-client/src/main/java/org/wildfly/extension/elytron/oidc/ProviderAttributeDefinitions.java

```diff
@@ -57,7 +61,7 @@ class ProviderAttributeDefinitions {
             new SimpleAttributeDefinitionBuilder(ElytronOidcDescriptionConstants.SSL_REQUIRED, ModelType.STRING, true)
                     .setAllowExpression(true)
                     .setDefaultValue(new ModelNode("external"))
-                    .setAllowedValues("external", "all", "none")
+                    .setValidator(EnumValidator.create(Oidc.SSLRequired.class, EnumSet.allOf(Oidc.SSLRequired.class)))
```

Comment:
FYI, I created https://github.com/wildfly/wildfly-core/pull/5427 to avoid the redundant set copy, after which this will just become:

```java
EnumValidator.create(Oidc.SSLRequired.class)